_includes/footer.html

```diff
@@ -5,7 +5,7 @@
     <label>Ready to hand off the Gruntwork?</label>
   </div>
   <div>
-    <button class="btn btn-primary-rounded">Buy Now</button>
+    <a href="/contact" class="btn btn-primary-rounded" role="button" aria-pressed="true">Buy Now</a>
```

Comment:
Why did this change?

Reply:
I had previously styled button links which I wanted to reuse. I can change it